pkg/tempopb/tempo.proto

```diff
@@ -360,6 +361,28 @@ message SpanMetricsResultPoint {
   uint64 exemplarDuration = 4;
 }
 
+message QueryInstantRequest {
+  string query = 1;
+  uint64 start = 2;
+  uint64 end = 3;
+}
+
+message QueryInstantResponse {
```

Comment:
Lean towards yes, but mainly because this is the route chosen by Prom/Mimir and also [Loki](https://grafana.com/docs/loki/latest/reference/loki-http-api/#query-logs-at-a-single-point-in-time).   Happy to tweak or change directions if we can think of other deciding factors - like this format impairs Grafana transformations or something?   One difference I see is that Prom/Loki still include the timestamp in the 1 sample response for instant queries. So it really boils down to instant returns `value: 1 sample`, range returns `values: [array of samples]`.

Reply:
no, i'm good with this. spoke with @mapno as well and agree for a different endpoint i am comfortable with a different return type.

i didn't want to impose the burden of more parsing/mapping/translating to grafana, but if we do this for all other data sources i'm good.